TOC.md

```diff
@@ -47,6 +47,7 @@ Explore all possible configuration scenarios and choose the right one for your e
 Even the best of us fail sometimes. We are working on it. 
 ## [Performance Counters Graphs](performance-counters-laptop-sleeping.md)
 ## [Scanning the AD](scan-ad-server-number.md)
+## [SQL log collection from a cluster](servers-in-cluster.md)
```

Comment:
I am not sure if there is a convention to always write cluster capitalized - SQL Cluster